src/olympia/amo/tests/__init__.py

```diff
@@ -171,19 +168,19 @@ def check_links(expected, elements, selected=None, verify=True):
         elif isinstance(item, str):
             text, link = None, item
 
-        e = elements.eq(idx)
+        elm = elements.eq(idx)
```

Comment:
why? Just curious.

Reply:
I hate single letter variables and tend to replace them whenever I see some (that were typically added 15 years ago). They often conflict with `ipdb` shortcuts and reduce readability for no benefit.